employeeService/util/db/employee_database_util.bal

```diff
@@ -60,10 +65,16 @@ public function updateData (string name, string age, string ssn, string employee
     // Initialize update status as unsuccessful MySQL operation
     json updateStatus = {"Status":"Data Not Updated"};
 
-    string sqlString = "UPDATE EMPLOYEES SET Name = '" + name + "', Age = '" + age + "', SSN = '" + ssn + "'WHERE
-                        EmployeeID  = '" + employeeId + "'";
+    // Prepare the sql string with employee data as parameters
+    sql:Parameter[] params = [];
+    sql:Parameter para1 = {sqlType:sql:Type.VARCHAR, value:name};
+    sql:Parameter para2 = {sqlType:sql:Type.VARCHAR, value:age};
+    sql:Parameter para3 = {sqlType:sql:Type.VARCHAR, value:ssn};
+    sql:Parameter para4 = {sqlType:sql:Type.VARCHAR, value:employeeId};
+    params = [para1, para2, para3, para4];
```

Comment:
Instead of two lines, shall we have `sql:Parameter[] params = [para1, para2, para3, para4];` . Same applies for the other two places as well

Reply:
Sure